modules/core/src/main/java/org/apache/ignite/configuration/ClientConfiguration.java

```diff
@@ -57,8 +57,11 @@ public final class ClientConfiguration implements Serializable {
     /** @serial Tcp no delay. */
     private boolean tcpNoDelay = true;
 
-    /** @serial Timeout. 0 means infinite. */
-    private int timeout;
+    /** @serial Connection timeout in milliseconds. 0 means infinite. */
+    private int connTimeout;
+
+    /** @serial Request timeout in milliseconds. 0 means infinite. */
```

Comment:
Please check that actual javadocs contains lines started with 'serial'. For which purpose this tag is used?